frontend/src/app/person-notes/person-notes.component.ts

```diff
@@ -31,10 +32,11 @@ export class PersonNotesComponent implements OnInit {
               private confirmService: ConfirmService) { }
 
   ngOnInit(): void {
-    const notes$ = this.personNoteService.list(this.person.id).share();
-    // display the spinner after 300ms, unless the notes have loaded before
-    Observable.of(true).delay(300).takeUntil(notes$).subscribe(() => this.spinnerDisplayed = true);
-    notes$.subscribe(notes => {
+    // display the spinner after 300ms, unless the notes have loaded before. Note: Observable.delay is untestable,
+    // see https://github.com/angular/angular/issues/10127
```

Comment:
Yup we had the same issue in the ebook and Pro Pack see https://github.com/Ninja-Squad/Formation/issues/217